packages/manager/apps/web/client/app/domain/webhosting/order/domain-webhosting-order-offer.class.js

```diff
@@ -0,0 +1,87 @@
+import capitalize from 'lodash/capitalize';
+import trim from 'lodash/trim';
+
+import {
+  DATABASE_ISOLATION_TYPES,
+  DISK_SIZE_MULTIPLE,
+  DISK_SIZE_UNIT,
+} from './domain-webhosting-order.constants';
+
+function getSpecificDatabases(databases, type) {
+  return databases.filter(({ isolation }) => isolation === type);
+}
+
+export default class WebHostingOffer {
+  constructor(properties) {
+    Object.assign(this, properties);
+
+    const sharedDatabases = this.databases
+      ? getSpecificDatabases(
+        this.databases,
+        DATABASE_ISOLATION_TYPES.SHARED,
+      )
+      : [];
+
+    this.sharedDatabases = sharedDatabases
+      .map(database => ({
+        available: database.available,
+        ...this.constructor.formatSizeValue(database.quota),
+        type: database.type,
+      }));
+  }
+
+  get localDatabases() {
+    const localDatabases = getSpecificDatabases(
+      this.databases,
+      DATABASE_ISOLATION_TYPES.LOCAL,
+    );
+
+    return localDatabases.map(({ engines }) => engines).flatten();
+  }
+
+  get formattedDiskSpace() {
+    return {
+      ...this.constructor.formatSizeValue(this.disk),
+      type: this.disk.type,
+    };
+  }
+
+  get formattedEmailAccount() {
+    return {
+      unit: this.emails.quota.unit,
+      value: `${this.emails.available} x ${this.emails.quota.value}`,
+    };
+  }
+
+  get formattedHighlight() {
+    return this.highlight ? capitalize(trim(this.highlight)) : this.highlight;
+  }
+
+  get formattedLanguages() {
+    const languagesKeys = Object.keys(this.languages);
+    const availableLanguages = languagesKeys
+      .filter(language => this.languages[language].length > 0);
+
+    return availableLanguages;
+  }
+
+  getSpecificDatabases(type) {
```

Comment:
It's a bit confusing because you also defined a function outside of this class that has the same name.

Is it intentional?

Reply:
I forgot to delete it